src/polkadot.scss

```diff
@@ -7,6 +7,23 @@ $btn-border-radius: 1.5rem;
 $btn-border-radius-lg: 2rem;
 $btn-border-radius-sm: 1.5rem;
 
+.card-button {
```

Comment:
We can also define the css classes directly in the index.css file in the src folder if they don't need to be compiled by sass.
./src/index.css